controllers/storagecluster/volumesnapshotterclasses.go

```diff
@@ -46,7 +46,7 @@ func newVolumeSnapshotClass(instance *ocsv1.StorageCluster, snapShotterType Snap
 		ObjectMeta: metav1.ObjectMeta{
 			Name: generateNameForSnapshotClass(instance, snapShotterType),
 		},
-		Driver: generateNameForSnapshotClassDriver(instance, snapShotterType),
+		Driver: generateNameForSnapshotClassDriver(snapShotterType),
```

Comment:
This will not work because the ocs-operator will never try to delete and recreate the class with the same name, some manual work is required after the upgrade to get the new snapshotclass. Are we fine with this one?

Reply:
@Madhu-1 It is okay, i just confirmed it.